go/sentry/localbackend/localbackend.go

```diff
@@ -0,0 +1,17 @@
+package localbackend
+
+import (
+	"context"
+
+	cmnGrpc "github.com/oasisprotocol/oasis-core/go/common/grpc"
+	"github.com/oasisprotocol/oasis-core/go/common/grpc/policy"
+	"github.com/oasisprotocol/oasis-core/go/sentry/api"
+)
+
+// LocalBackend is a local sentry backend implementation.
```

Comment:
The interface should be part of the general API (e.g., `go/sentry/api`), why the move here?

Reply:
Because it caused import cycle hell and this was the cleanest way to solve it.  This might improve after moving the policy watcher to the sentry worker, will revisit this later.

Reply:
Ok, if nothing else it should at least be under `go/sentry/api/local`.

Reply:
OK, after moving the policy watcher, I was able to move this back to `go/sentry/api` :)